requirements-docs.txt

```diff
@@ -1 +1,3 @@
 Sphinx==1.3.1
+sphinx-django-command
```

Comment:
Btw. it's possible to use sphinx-apidoc to build docs and include them like this:

```

.. automodule:: dbbackup.settings
  :undoc-members:
  :show-inheritance: